src/opt/vyatta/share/vyatta-op/templates/clear/interfaces/connection/node.tag/node.def

```diff
@@ -0,0 +1,53 @@
+help:  Bring datalink layer connection down and back up
+
+allowed: local -a array ;
+         array=( /etc/ppp/peers/pppoe* /etc/ppp/peers/pppoa* /etc/ppp/peers/wan* /etc/ppp/peers/wlm* ) ;
```

Comment:
`/etc/ppp/peers/wan*` is irrelevant since we don't have serial WAN support now.

Can `wlm*` interfaces really be restarted this way?

Reply:
Added by accident - rewritten by proper XML logic. Thanks!